protocol-definitions/Jet.yaml

```diff
@@ -416,3 +416,28 @@ methods:
           nullable: false
           since: 2.6
           doc: 'True if processed successfully, false otherwise'
+  - id: 21
```

Comment:
is there a reason we skip id 19 and 20 and jump from 18 to 21 here?

Reply:
I assume this is because we use those ids in https://github.com/hazelcast/hazelcast-client-protocol/pull/447. So, I am resolving this, assuming we will merge that PR before 5.3 as well